base/linalg/symmetric.jl

```diff
@@ -64,6 +64,8 @@ issymmetric{T<:Complex,S}(A::Hermitian{T,S}) = all(imag(A.data) .== 0)
 issymmetric(A::Symmetric) = true
 transpose(A::Symmetric) = A
 ctranspose{T<:Real}(A::Symmetric{T}) = A
+isposdef(A::Symmetric) = isposdef(A.data)
```

Comment:
I tried allowing more types to the signature but this did not work
```julia
function isposdef{Tv<:VTypes}(A::Union{SparseMatrixCSC{Tv,SuiteSparse_long}, Symmetric})
    if !ishermitian(A)
        return false
    end
    try
        f = cholfact(A)
    catch e
        isa(e, LinAlg.PosDefException) || rethrow(e)
        return false
    end
    true
end
```
and also there is a extra check `ishermitian` which is not required. I have pushed a commit with changes, I am not sure if this is the optimal way of doing things to add such a feature but any comment would be helpful. The reference for this is the following : understanding of [this](https://en.wikipedia.org/wiki/Positive-definite_matrix) wiki article, which gives a iff condition for a matrix being positive definite in relation to Cholesky decomposition.